components/cam/bld/build-namelist

```diff
@@ -3499,6 +3499,8 @@ add_default($nl, 'micro_p3_tableversion');
 add_default($nl, 'micro_aerosolactivation');
 add_default($nl, 'micro_subgrid_cloud');
 add_default($nl, 'micro_tend_output');
+add_default($nl, 'p3_QcAutoCon_Expon');
```

Comment:
At some point, it would be nice if we settled on a consistent style for naming variables and symbols in general. Mixing "camelCase" and "snake_case" and "bothCamel_and_snakeCase" isn't too harmful, but it's kind of needlessly distracting. Maybe I'm the only one who cares about this kind of stuff, though.